packages/chakra-ui/src/ChakraFrameProvider.tsx

```diff
@@ -24,14 +24,16 @@ const memoizedCreateCacheWithContainer = weakMemoize((container: HTMLElement) =>
   return newCache;
 });
 
+// const system = createSystem(defaultConfig, { preflight: false });
+
 export const __createChakraFrameProvider =
   (props: any) =>
   ({ document }: any) => {
     return (
       <div style={{ margin: 2 }}>
         <CacheProvider value={memoizedCreateCacheWithContainer(document.head)}>
-          <ChakraProvider resetCSS={false}>
-            <CSSReset />
+          <ChakraProvider value={defaultSystem}>
+            {/* <CSSReset /> TODO: figrue out styling issues */}
```

Comment:
I'm assuming you'll need to fix this before merging?

Reply:
Yes they do look correct without it.